pkg/acp/client.go

```diff
@@ -26,12 +29,14 @@ func NewClient(serverCertPath string, certPath string, keyPath string, cfg Confi
 	clientCertPool := x509.NewCertPool()
 	clientCertPool.AppendCertsFromPEM(clientCACert)
 
+	// It assigns a pool with certificates to the HTTP client.
```

Comment:
Drop the `it assigns` and change to `assign a pool...` - it's gonna be more consistent with comments in other files.